lib/react_on_rails/configuration.rb

```diff
@@ -123,6 +124,18 @@ def setup_config_values
 
     private
 
+    def check_auto_load_requirements_if_auto_load
+      return if auto_load_bundle != true
+
+      raise_missing_components_subdirectory unless components_subdirectory.present?
```

Comment:
Only continue checks if this one is present.

Throw if auto_load_bundle is true.

order matters, b/c it's fine for auto_load_bundle **DEFAULT** to be false and used only in a few places.